app/controllers/users_controller.rb

```diff
@@ -5,6 +5,7 @@ class UsersController < ApplicationController
   before_action :check_recent_authentication!, only: %i[enable_2fa disable_2fa regenerate_2fa_backup_codes]
   before_action :set_recent_authentication!, only: %i[edit update enable_2fa disable_2fa]
   before_action -> { redirect_to_root_unless_user(:can_admin_results?) }, only: %i[admin_search]
+  before_action -> { redirect_to_root_unless_user(:can_admin_results?) }, only: %i[merge]
```

Comment:
If I'm not mistaken, then these two `before_action` bodies are completely similar. You can merge them into one if you use `only: %i[admin_search merge]`

Reply:
Oh yes, thank you